components/runners/ambient-runner/ambient_runner/bridges/claude/mcp.py

```
@@ -272,3 +292,68 @@ def check_mcp_authentication(server_name: str) -> tuple[bool | None, str | None]
 
     return None, None
+
+
+# ---------------------------------------------------------------------------
+# Gerrit MCP config generation
+# ---------------------------------------------------------------------------
+
+
+def generate_gerrit_config(instances: list[dict]) -> None:
+    """Generate Gerrit MCP server configuration from credential instances."""
+    config_dir = Path("/tmp/gerrit-mcp")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/gerrit-mcp"

(S108)